newlib/libc/sys/hermit/unistd.c

```diff
@@ -13,6 +13,10 @@ void _exit(int status) {
 	sys_exit(status);
 }
 
+unsigned alarm(unsigned __secs) {
```

Comment:
Since we cannot provide an error programmatically, I am wondering if we should print a warning when this function is used:

```c
fprintf(stderr, "stubbed alarm() called. SIGALRM will not be generated!\n");
```